envoy/formatter/substitution_formatter.h

```diff
@@ -100,9 +100,9 @@ class HttpFormatterContext {
   AccessLog::AccessLogType accessLogType() const;
 
 private:
-  const Http::RequestHeaderMap* request_headers_{};
-  const Http::ResponseHeaderMap* response_headers_{};
-  const Http::ResponseTrailerMap* response_trailers_{};
+  mutable const Http::RequestHeaderMap* request_headers_{};
+  mutable const Http::ResponseHeaderMap* response_headers_{};
+  mutable const Http::ResponseTrailerMap* response_trailers_{};
```

Comment:
Are these changes necessary? I wouldn't think so since their private and we're not changing the const-ness of any methods (or adding any new const methods).

Reply:
We may update them in the requestHeaders/responseHeaders.